app/views/users/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1><%= current_user.first_name %>'s Folio</h1>
+
+<table>
+  <tr>
+    <td>
+      <%= link_to "Create New Folder", new_folder_path(@current_folder), class: "btn" %>
```

Comment:
Do we have any opinions on moving the folder form onto the folder page? I just think it's odd that the upload form is on the page, but the folder form is on a different page. I think we should either have both the forms on the page or both the links to the forms on the page.